ark/segmentation/signal_extraction.py

```diff
@@ -36,16 +39,20 @@ def center_weighting_extraction(cell_coords, image_data, centroid):
             values representing pixels within one cell
         image_data (xarray.DataArray):
             array containing channel counts
-        centroid (tuple):
-            the centroid of the region in question
+        **kwargs:
+            arbitrary keyword arguments
 
     Returns:
         numpy.ndarray:
             Sums of counts for each channel
     """
 
+    # check for centroid in kwargs
+    if 'centroid' not in kwargs.keys():
```

Comment:
This won't ever happen though, right? There's no user input here, it gets called internally

Reply:
True.  Just auto-piloted that one haha. Thought it might help catch bugs if we end up dev-ing but accessing a non-existent argument in a dictionary already throws an error, so it's just redundant.